RadixWalletTests/Clients/TokenPriceCientTests/TokenPriceCientTests.swift

```diff
@@ -19,47 +20,45 @@ final class TokenPriceCientTests: XCTestCase {
 	}
 
 	func test_withDecimalPlaces_1() {
-		validateDecimalPriceConversion(1.99, expected: try! .init(value: "1.99"))
+		validateDecimalPriceConversion(1.99, expected: try! Decimal192("1.99"))
 	}
 
 	func test_withDecimalPlaces_2() {
-		validateDecimalPriceConversion(1.000099, expected: try! .init(value: "1.000099"))
+		validateDecimalPriceConversion(1.000099, expected: try! Decimal192("1.000099"))
 	}
 
 	func test_belowOne_1() {
 		validateDecimalPriceConversion(0.99, expected: 0.99)
 	}
 
 	func test_belowOne_2() {
-		validateDecimalPriceConversion(0.000099, expected: try! .init(value: "0.000099"))
+		validateDecimalPriceConversion(0.000099, expected: try! Decimal192("0.000099"))
 	}
 
-	// NOTE: All of the below values would be rounded to 14 decimal places
-	//       As it seems that Swift number formatter for Double cannot express more decimals.
-	func test_closeToRETDecimalDivisibility() {
+	func test_closeToDecimal192Divisibility() {
```

Comment:
@GhenadieVP @kugel3 we just got two more decimal places of precison, 16 instead of 14, for these tests, Sargon is wise! :)